pom.xml

```diff
@@ -664,6 +664,41 @@
                     </replacementProperties>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>dk.kb.util</groupId>
+                <artifactId>injection-maven-plugin</artifactId>
+                <version>1.0.0</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>read-yaml-properties</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <filePath>conf/ds-datahandler-behaviour.yaml</filePath>
+                    <yamlResolvers>
+                        <YamlResolver>
+                            <yamlType>List</yamlType>
+                            <yamlPath>oai_targets</yamlPath>
+                            <sourceKey>name</sourceKey>
+                            <destinationKey>oai_targets</destinationKey>
+                            <createEnum>true</createEnum>
+                        </YamlResolver>
+                        <yamlResolver>
+                            <yamlType>List</yamlType>
+                            <yamlPath>present.origins</yamlPath>
+                            <destinationKey>present_origins</destinationKey>
+                            <createEnum>true</createEnum>
+                        </yamlResolver>
+                        <yamlResolver>
+                            <yamlType>Single-value</yamlType>
+                            <yamlPath>present.origins[7]</yamlPath>
```

Comment:
Not too happy about `present.origins[7]` as it relies on the full list of `origins` being unchanged. Some ideas:

1. Make an explicit `default_origin` in `ds-datahandler-behaviour.yaml`
2. Use `present.origins[last]` (and make a comment in the config about which value will be the default)
3. Reorder the `origins` and use `present.origins[0]` (and make a comment as above)